scRNA-seq-advanced/setup/intro-citeseq-setup.Rmd

```diff
@@ -0,0 +1,410 @@
+
+- `CD3`
+  - "CD3 (cluster of differentiation 3) is a protein complex and T cell co-receptor that is involved in activating both the **cytotoxic T cell (CD8+ naive T cells) and T helper cells (CD4+ naive T cells).** ... This high specificity, combined with the presence of CD3 at all stages of T-cell development, makes it a useful immunohistochemical marker for T-cells in tissue sections."  [source](https://en.wikipedia.org/wiki/CD3_(immunology))
+- `CD4`
+  - TCR co-receptor that we can probably assume is mostly on CD4+ T-cells
+- `CD8`
+  - TCR co-receptor that we can probably assume is mostly on CD8+ T-cells
+- `CD11c`
+  - "CD11c is a type I transmembrane protein that is expressed on **monocytes**, **granulocytes**, a subset of **B cells**, **dendritic cells**, and **macrophages** and may be demonstrated on B-cell chronic lymphocytic leukemia, marginal zone lymphomas, and hairy cell leukemia." [source](https://www.sciencedirect.com/topics/neuroscience/cd11c)
+- `CD14`
+  - "CD14 is a glycolipid-anchored membrane glycoprotein expressed on cells of the myelomonocyte lineage including **monocytes**, **macrophages**, and some **granulocytes**." [source](https://www.sciencedirect.com/topics/neuroscience/cd14)
+- `CD16`
+  - "CD16 is involved in antibody-dependent cell-mediated cytotoxicity and is expressed on large granular lymphocytes (LGLs) of both **NK- and T-cell types**." [source](https://www.sciencedirect.com/topics/neuroscience/cd16)
+- `CD19`
+  - "CD19 is an IgSF surface glycoprotein of 95 kDa that is expressed from the earliest stages of **B cell** development until plasma cell terminal differentiation, when its expression is lost." [source](https://www.sciencedirect.com/topics/neuroscience/cd19)
+- `CD56`
+  - "CD56 expression is also found in, among others, the hematopoietic system. Here, the expression of CD56 is mostly associated with, but not limited to, **natural killer cells**. CD56 has been detected on other lymphoid cells, including **gamma delta Τ cells and activated CD8+ T cells**, as well as on **dendritic cells**." [source](https://en.wikipedia.org/wiki/Neural_cell_adhesion_molecule)
+
+
+
+Let's color cells by their most prevalent ADT:
+
+```{r does anybody have a UMAP}
+scater::plotReducedDim(norm_sce, "UMAP", colour_by = "max_adt", 
+                       point_alpha = 0.6, point_size = 0.25)
+```
+
+
+
+
+## Cell type annotation from RNA
+
+We can introduce annotation with `SingleR`, and it might be a good idea to show how the results are highly influenced by the reference. 
+For now, I'm just using the `HumanPrimaryCellAtlasData` reference as a starting place.
+Note that all references here involve some caching, and we can expect some interactive prompting to go by for setting up these caches that we'll have to walk everyone through.
+
+
+```{r}
+ref_data <- celldex::HumanPrimaryCellAtlasData(ensembl = TRUE) # pick one for now to start
+preds <- SingleR::SingleR(
```

Comment:
```suggestion
singleR_predictions <- SingleR::SingleR(
```
super minor, but can we name this something slightly more informative? I'm open to other names too like `celltype_predictions`